### PR TITLE
Add Austrian district data to knowledge_graph.csv and clean up file.

### PR DESCRIPTION
**Pull request by geening**

### CODE REVIEW

File: src/data/knowledge_graph.csv [hunk 1]
```diff
@@ -680,23 +680,117 @@ AR_Z_049,Q615066,
 AS,Q16641,country/ASM
 AT,Q40,country/AUT
 AT_1,Q43210,ISO/AT-1
+AT_1_101,Q126321,wikidataId/Q126321
```
Comment (owahltinez) [inline src/data/knowledge_graph.csv:683]: If we know the NUTS code, we prefer to use `nuts/${code}` (even if the entry does not exist in DataCommons yet)

**Review by owahltinez — CHANGES_REQUESTED** [review]
One small request (replace `wikidataId/*` with `nuts/*`) but other than that LGTM. Thanks for cleaning up the out-of-order entries!

File: src/data/knowledge_graph.csv (hunk 1 above)
Comment (geening) [reply src/data/knowledge_graph.csv:683]: We know the NUTS codes for the six districts that correspond precisely to NUTS regions.  Should I change those to use nuts/${code} and use wikidataId/${code} for the others?

**Comment by owahltinez** [comment]
Right, I forgot these weren't NUTS. In that case, I suggest switching the
ones that we know are NUTS to nuts/* and leave the rest as null.

File: src/data/knowledge_graph.csv (hunk 1 above)
Comment (geening) [reply src/data/knowledge_graph.csv:683]: Any particular reason why we don't want to include the wikidataId/${code} for the districts that are not NUTS?

Comment (geening) [reply src/data/knowledge_graph.csv:683]: wikidataId does show up in datacommons, e.g., https://datacommons.org/place/wikidataId/Q13298

**Comment by owahltinez** [comment]
It's redundant information. You can trivially derive that from the Wikidata
code itself.

Sadly, we had to give up on being able to find matches in DataCommons for
the majority of regions.

**Comment by geening** [comment]
Ok, yes a human can certainly trivially derive the datacommons url.  I just wasn't sure what the file was used for -- is it ever used in an automated fashion?

**Comment by owahltinez** [comment]
It is not used automatically. Even if it was, when a location can only be
found in DataCommons by its Wikidata ID, it doesn't have any information we
don't already have. That's why we prefer any identifier other than Wikidata
ID, because it indicates that DataCommons might have some other information
associated with the location.

Since a location which only has Wikidata ID doesn't help in any way, we'd
rather have null.

**Comment by geening** [comment]
Wikidata IDs removed from datacommons column, and NUTS identifiers inserted when available.

**Review by owahltinez — APPROVED** [review]
LGTM, thank you!

**Pull request merged**